oap-server/server-receiver-plugin/aws-firehose-receiver/src/main/java/org/apache/skywalking/oap/server/receiver/aws/firehose/AWSFirehoseReceiverModuleProvider.java

```diff
@@ -0,0 +1,105 @@
+    public static final String NAME = "default";
+
+    private AWSFirehoseReceiverModuleConfig moduleConfig;
+    private HTTPServer httpServer;
+
+    @Override
+    public String name() {
+        return NAME;
+    }
+
+    @Override
+    public Class<? extends ModuleDefine> module() {
+        return AWSFirehoseReceiverModule.class;
+    }
+
+    @Override
+    public ConfigCreator<AWSFirehoseReceiverModuleConfig> newConfigCreator() {
+        return new ConfigCreator<AWSFirehoseReceiverModuleConfig>() {
+            @Override
+            public Class<AWSFirehoseReceiverModuleConfig> type() {
+                return AWSFirehoseReceiverModuleConfig.class;
+            }
+
+            @Override
+            public void onInitialized(final AWSFirehoseReceiverModuleConfig initialized) {
+                moduleConfig = initialized;
+            }
+        };
+    }
+
+    @Override
+    public void prepare() throws ServiceNotProvidedException {
+        final HTTPServerConfig httpServerConfig = HTTPServerConfig.builder().host(moduleConfig.getHost())
+                                                                  .port(moduleConfig.getPort())
+                                                                  .contextPath(moduleConfig.getContextPath())
+                                                                  .maxThreads(moduleConfig.getMaxThreads())
+                                                                  .idleTimeOut(moduleConfig.getIdleTimeOut())
+                                                                  .acceptQueueSize(moduleConfig.getAcceptQueueSize())
+                                                                  .maxRequestHeaderSize(
+                                                                      moduleConfig.getMaxRequestHeaderSize())
+                                                                  .enableTLS(moduleConfig.isEnableTLS())
+                                                                  .tlsKeyPath(moduleConfig.getTlsKeyPath())
+                                                                  .tlsCertChainPath(moduleConfig.getTlsCertChainPath())
+                                                                  .enableContentEncoding(true)
+                                                                  .build();
+        httpServer = new HTTPServer(httpServerConfig);
+        httpServer.initialize();
+    }
+
+    @Override
+    public void start() throws ServiceNotProvidedException, ModuleStartException {
+        final OpenTelemetryMetricRequestProcessor processor = getManager().find(OtelMetricReceiverModule.NAME)
+                                                                          .provider()
+                                                                          .getService(
+                                                                              OpenTelemetryMetricRequestProcessor.class);
```

Comment:
When https://github.com/apache/skywalking/pull/10300/files#r1083433476 happens, you might get an implementation of `OpenTelemetryMetricRequestProcessor` that you want